pages/roadmap.js

```diff
@@ -0,0 +1,76 @@
+import GenericLayout from '../components/layout/GenericLayout'
+import IconDocuments from '../components/icons/Documents'
+import IconCode from '../components/icons/Code'
+import IconPowerPlug from '../components/icons/PowerPlug'
+import GeneratorInstallation from '../components/GeneratorInstallation'
+
+export default function RoadmapPage() {
+  const description = 'Long-term vision and plans for the AsyncAPI Initiative'
+  const image = 'Image for meta tags here...'
+  // const image = '/img/social/generator.png'
```

Comment:
Shall this be removed in the end doc?

Reply:
This will be replaced with a proper image, that's why it's left there.